docs/miscellaneous.rst

```diff
@@ -394,12 +394,14 @@ The following is the order of precedence for operators, listed in order of evalu
 +============+=====================================+============================================+
 | *1*        | Postfix increment and decrement     | ``++``, ``--``                             |
 +            +-------------------------------------+--------------------------------------------+
-|            | Function-like call                  | ``<func>(<args...>)``                      |
+|            | New expression                      | ``new <typename>``                         |
 +            +-------------------------------------+--------------------------------------------+
 |            | Array subscripting                  | ``<array>[<index>]``                       |
 +            +-------------------------------------+--------------------------------------------+
 |            | Member access                       | ``<object>.<member>``                      |
 +            +-------------------------------------+--------------------------------------------+
+|            | Function-like call                  | ``<func>(<args...>)``                      |
```

Comment:
I think we also have to include the associativity direction and the fact that `new <typename>` might actually be `new <expression>`.